core/libraries/batch/JobHandlers/PreviewEventDeletion.php

```diff
@@ -226,7 +226,7 @@ public function continue_job(JobParameters $job_parameters, $batch_size = 50)
         $job_parameters->mark_processed($units_processed);
         // If the most-recently processed root node is complete, we must be all done because we're doing them
         // sequentially.
-        if (isset($root_node) && $root_node instanceof ModelObjNode && $root_node->isComplete()) {
+        if (! isset($root_node) || (isset($root_node) && $root_node instanceof ModelObjNode && $root_node->isComplete())) {
```

Comment:
hmmmm....

would be better if this displayed a warning/error notice that nothing was selected for deletion but that's not a quick fix, so we'll ride with this for the time being, but I'm going to create an issue for improving this UX